Permission and Access Helper/force-app/main/default/classes/PermissionController.cls

```diff
@@ -0,0 +1,33 @@
+public class PermissionController {
+    
+    @AuraEnabled(cacheable=true)
+    public static List<ComboboxDataWrapper> getProfile(){
+        return PermissionService.getProfile();
```

Comment:
put  PermissionService.getProfile(); in try catch and throw AuraHandledException  if any exception occurs

Reply:
same for all controller methods

Reply:
Done